lua/gk.lua

```diff
@@ -48,6 +48,10 @@ return function (net_conf, sol_conf, gk_lcores)
 	gk_conf.back_max_pkt_burst =
 		gatekeeper.get_back_burst_config(
 			gk_max_pkt_burst_back, net_conf)
+	if lls_conf.lls_max_pkt_burst < math.max(gk_conf.front_max_pkt_burst,
+			gk_conf.back_max_pkt_burst) then
+		error("Wrong packet burst size configurations for LLS and GK")
```

Comment:
Since ``run_lls()`` in ``lua/lls.lua`` will be called before ``lua/gk.lua``, we have already initialized LLS' mailbox with the given ``ls_conf.lls_max_pkt_burst`` before. Even if we change the value of ``ls_conf.lls_max_pkt_burst`` here, it won't change the size of the entries in LLS mailbox.

Reply:
You've just found the reason why the initialization of the mailbox of the LLS block must be done in `lls_stage1()`. Please add a comment in `lls_stage1()` explaining why the mailbox is being initialized there.

Reply:
Once you do the change I describe in my previous comment, update the comment in file `lua/lls.lua` before line `lls_conf.lls_max_pkt_burst = 32`.